submissions/ayotolaTehingbola/htmlForms/signupPage.html

```diff
@@ -0,0 +1,132 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta
+      name="keywords"
+      content="create account, new customer, signup, new account"
+    />
+    <meta name="description" content="Sign Up Page" />
+    <meta name="author" content="Ayotola Tehingbola" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link rel="stylesheet" type="text/css" href="../css/stylesheet.css" />
+    <title>Create new Account</title>
+  </head>
+
+  <body class="container">
+    <div class="header">
+      <header>
+        <h2>UNBRANDED</h2>
+      </header>
+
+      <nav class="navbar">
+        <ul class="navbar-list">
+          <li><a href="#"> Shop </a></li>
+          <li><a href="#"> Contact </a></li>
+          <li><a href="./signupPage.html"> Sign Up </a></li>
+        </ul>
+      </nav>
+    </div>
+
+    <div class="inner-container">
+      <div class="main-block">
+        <h3>Create An Account</h3>
+        <p>Join Unbranded now and start shopping straight away!</p>
+        <br /><br />
+        <p>
+          Already got an account?
+          <a href="#" id="existing-account-signin"> Sign In </a>
+        </p>
+      </div>
+
+      <div class="side-bar">
+        <!--Signup Form-->
+        <form
+          class="form"
+          action="#"
```

Comment:
@ayotola-t Remove the attributes `action`, `method` and `enctype`in the form element here and below. They are not needed yet.